custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -17,9 +17,9 @@
 TEST_CASE_TYPE = 'test'
 
 
-def validate_number(string_value):
+def validate_phone_number(string_value):
     if string_value is None or string_value.strip() == '':
-        return None
+        return ''
     else:
         return int(string_value)
```

Comment:
It looks like all of the numbers in the data have 10 digits (leading digit non-zero).  Would it be sufficient to assert that the returned string has 10 characters?  Noting that if we have to integrate with messaging proper formatting would be required.